examples/bert/bert_cuda_gpu.json

```diff
@@ -76,7 +76,6 @@
             }
         },
         "evaluator": "common_evaluator",
-        "execution_providers": ["CUDAExecutionProvider"],
```

Comment:
target it not specified here. so, this would also loop over the installed eps.

Reply:
yes, it will use the installed the EPs.

Reply:
for this example, we need a target then? Otherwise, the example is not the same as before.

Reply:
we can make the examples configure as simple as possible. This is suggested from @devang-ml. 

IMHO, basically, so long as the example could works. the config for hardware doesn't matter.